operations/iuf/workflows/product_delivery.md

```diff
@@ -56,11 +56,26 @@ Refer to that table and any corresponding product documents before continuing to
 
 1. Invoke `iuf run` with activity identifier `${ACTIVITY_NAME}` and use `-r` to execute the [`deliver-product`](../stages/deliver_product.md) stage. Perform the upgrade using product content found in `${MEDIA_DIR}`.
 
-    (`ncn-m001#`) Execute the `deliver-product` stage.
+    1. Option 1: Install or upgrade without SLURM or PBS Products
 
-    ```bash
-    iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run -r deliver-product
-    ```
+       (`ncn-m001#`) Execute the `deliver-product` stage.
+
+       ```bash
+       iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run -r deliver-product
+       ```
+
+    1. Option 2: Install or upgrade along with SLURM or PBS Products
```

Comment:
SLURM and PBS Pro are technically included in the USS product starting in version 1.1.   So we might want to make that clearer in the description.   

`Option 2:  Enable SLURM or PBS Pro Products when installing USS 1.1 or higher`